extensions/display-modes/src/index.tsx

```diff
@@ -0,0 +1,115 @@
+  }
+
+  function formatDisplayKind(kind: DisplayKind): string {
+    switch (kind) {
+      case DisplayKind.builtIn:
+        return "Built-in";
+      case DisplayKind.external:
+        return "External";
+      default:
+        return "Unknown";
+    }
+  }
+
+  function formatDisplayMode(mode: Mode): string {
+    const widthFormatted = mode.width.toString().padStart(5, " ");
+    const heightFormatted = mode.height.toString().padStart(5, " ");
+    const scaleFormatted = mode.scale?.toString().padStart(2, " ") ?? "N/A";
+    const refreshRateFormatted = mode.refreshRate.toString().padStart(4, " ");
+
+    return `${widthFormatted} x ${heightFormatted} @ ${scaleFormatted}x ${refreshRateFormatted}Hz`;
+  }
+
+  const navigation = useNavigation();
+
+  function detail(display: DisplayInfo): ReactNode {
+    // First, map the modes to an array of objects with mode and isEqual properties
+    const modesWithComparison = display.modes.map((mode) => ({
+      mode: mode,
+      isEqual: areModesEqual(mode, display.currentMode),
+    }));
+
+    return (
+      <List navigationTitle={formatTitle(display)}>
+        {modesWithComparison.map((modeInfo, index) => {
+          const key = numberToKey(index + 1);
+          return (
+            <List.Item
+              key={index}
+              icon={modeInfo.isEqual ? "selected-mode-icon.png" : "mode-icon.png"}
+              title={formatDisplayMode(modeInfo.mode)}
+              subtitle={modeInfo.isEqual ? "Current Mode" : undefined}
+              actions={
+                <ActionPanel>
+                  <Action
+                    title="Change Display Mode"
+                    {...(key ? { modifiers: ["opt"], key: key } : {})}
+                    onAction={async () => {
+                      const result = await setMode(display.display.id, modeInfo.mode);
+
+                      if (result) {
+                        await showHUD(`Mode changed successfully ${formatDisplayMode(modeInfo.mode)}`);
+                        await fetchDisplaysInfo();
+                        navigation.pop();
+                      } else {
+                        await showHUD("Failed to change display mode");
+                      }
+                    }}
+                  />
+                </ActionPanel>
+              }
+            />
+          );
+        })}
+      </List>
+    );
+  }
+
+  return (
+    <List isLoading={displays === undefined}>
+      {displays?.map((display, index) => {
+        const key = numberToKey(index + 1);
+        return (
+          <List.Item
+            key={index}
+            icon="display-icon.png"
+            title={formatTitle(display)}
+            subtitle={formatSubtitle(display)}
+            actions={
+              <ActionPanel>
+                <Action.Push
+                  title="Show Display Modes"
+                  target={detail(display)}
+                  {...(key ? { modifiers: ["opt"], key: key } : {})}
```

Comment:
```suggestion
                  {...(key ? { shortcut: { modifiers: ["opt"], key: key } } : {})}
```
You missed `shortcut` here, it should look like the above suggestion

If you hold  `⌘` for 400 ms you'll get the same behavior out of the box, I would actually just remove the above and rely on  `⌘+num` instead which is how Raycast works in every list 🙂

Reply:
Oh, yeah this is totally unnecessary then. I removed it.
Very convenient default behaviour from Raycast 👏